src/scripts/main.js

```diff
@@ -1,3 +1,336 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const table = document.querySelector('table');
+const th = document.querySelectorAll('th');
+let previous = '';
+
+for (const cell of th) {
+  cell.setAttribute('data-Asc', true);
+}
+
+function toNumber(string) {
+  const number = string.slice(1).split('').filter(el => el !== ',').join('');
+
+  return +number;
+}
+
+function stringCompare(a, b, child) {
+  const result
+  = a.children[child].textContent.localeCompare(b.children[child].textContent);
+
+  return result;
+}
+
+// Sorting function
+
+const sort = (ev) => {
+  const tr = document.querySelectorAll('tr');
+  const tableContent = [...tr].slice(1, tr.length - 1);
+
+  tbody.innerHTML = '';
+
+  if (ev.target.dataset.asc === 'true'
+  || (ev.target.dataset.asc === 'false' && previous !== ev.target.innerText)) {
+    switch (ev.target.innerText) {
+      case 'Name':
+        tableContent.sort((a, b) =>
+          stringCompare(a, b, 0)
+        );
+        break;
+      case 'Position':
+        tableContent.sort((a, b) =>
+          stringCompare(a, b, 1)
+        );
+        break;
+      case 'Office':
+        tableContent.sort((a, b) =>
+          stringCompare(a, b, 2)
+        );
+        break;
+      case 'Age':
+        tableContent.sort((a, b) =>
+          a.children[3].textContent - b.children[3].textContent
+        );
+        break;
+      case 'Salary':
+        tableContent.sort((a, b) =>
+          toNumber(a.children[4].textContent)
+           - toNumber(b.children[4].textContent)
+        );
+        break;
+      default:
+        break;
+    }
+
+    ev.target.dataset.asc = 'false';
+    previous = ev.target.innerText;
+  } else if (
+    ev.target.dataset.asc === 'false' && previous === ev.target.innerText) {
+    switch (ev.target.innerText) {
+      case 'Name':
+        tableContent.sort((a, b) =>
+          stringCompare(b, a, 0)
```

Comment:
You don't need another if statement here. Think how to do it without it using a ternary operator